cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -662,6 +663,84 @@ public void checkCustomCaSecret(CertificateAuthority ca, Secret certSecret, Secr
             return Future.succeededFuture(this);
         }
 
+        Future<ReconciliationState> zkVersionChange() {
+
+            String kafkaSsName = KafkaCluster.kafkaClusterName(name);
+            String zkSsName = ZookeeperCluster.zookeeperClusterName(name);
+
+            Future<StatefulSet> kafkaSsFuture = kafkaSetOperations.getAsync(namespace, kafkaSsName);
+            Future<StatefulSet> zkSsFuture = zkSetOperations.getAsync(namespace, zkSsName);
+
+            return CompositeFuture.all(kafkaSsFuture, zkSsFuture).compose(allFutures -> {
+                if (allFutures.succeeded()) {
+
+                    StatefulSet zkSs = zkSsFuture.result();
+                    StatefulSet kafkaSs = kafkaSsFuture.result();
```

Comment:
`allFutures.resultAt(0)` and `allFutures.resultAt(1)` does the same thing, and will allow you to inline the `zkSsFuture` and `kafkaSsFuture` variables.